pkg/console/starter/starter.go

```diff
@@ -252,13 +252,13 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 
 	staleConditionsController := staleconditions.NewRemoveStaleConditions(
```

Comment:
I'd delete or maybe comment out this whole block and `staleConditionsController.Run` if we're not using it.

Reply:
agree, dont think this is necessary anymore :)

Reply:
I can comment it out, good with that.
With an explicit reminder comment about re-enabling the `Run()`, that tends to be a gotcha.